tests/build.sh

```diff
@@ -2,14 +2,14 @@
 
 mkdir -p bin/temp
 
-for f in *.s
+for f in *.s */*.s
 do
   echo "Assembling $f file..."
   # take action on each file. $f store current file name
   OBJECTFILE="bin/temp/$(basename "$f" .s).o"
   FINALFILE="bin/$(basename "$f" .s)"
-  arm-none-eabi-as -march=armv6-m -o $OBJECTFILE $f
+  arm-none-eabi-as -march=armv6s-m -o $OBJECTFILE $f
```

Comment:
ACK (for public knowledge, armv6-m targeting for some reason disallows the SVC instruction. Most Cortex-M0 etc projects use armvs-m to workaround this limitation)

Reply:
Added a code comment about the reason for the switch